src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -4,171 +4,120 @@
             [quo2.core :as quo]
             [quo2.foundations.colors :as colors]
             [react-native.core :as rn]
+            [clojure.string :as string]
             [react-native.platform :as platform]
             [reagent.core :as reagent]
             [status-im2.common.constants :as constants]
             [status-im2.common.scroll-page.view :as scroll-page]
             [status-im2.contexts.communities.overview.style :as style]
             [status-im2.contexts.communities.menus.community-options.view :as options]
             [status-im2.contexts.communities.menus.request-to-join.view :as join-menu]
+            [status-im2.contexts.communities.overview.mocks :as mocks]
+            [cljs.core :as core]
             [utils.re-frame :as rf]))
 
-(def knc-token-img (js/require "../resources/images/tokens/mainnet/KNC.png"))
-(def mana-token-img (js/require "../resources/images/tokens/mainnet/MANA.png"))
-(def rare-token-img (js/require "../resources/images/tokens/mainnet/RARE.png"))
-(def eth-token-img (js/require "../resources/images/tokens/mainnet/ETH.png"))
-(def dai-token-img (js/require "../resources/images/tokens/mainnet/DAI.png"))
 
-;; Mocked list items
-(def user-list
-  [{:full-name "Alicia K"}
-   {:full-name "Marcus C"}
-   {:full-name "MNO PQR"}
-   {:full-name "STU VWX"}])
+(defn join-existing-users-string
```

Comment:
This is some heavy logic in the view ;)

I think this function should be either a subscription or some utility function that can live a local `utils` file or directory. This idea is documented in the Project Structure section of the guidelines (https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#project-structure)

Having the subscription or localized utility function will also help us write a unit test for this function. It's begging for tests ;)